packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -593,7 +593,6 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   bool _wasSelectingVerticallyWithKeyboard = false;
 
   void _setTextEditingValue(TextEditingValue newValue, SelectionChangedCause cause) {
-    textSelectionDelegate.textEditingValue = newValue;
```

Comment:
I kept this for backward compatibility, we should only remove it once we remove the deprecated setter textEditingValue

Reply:
Could you elaborate? People are required to implement `userUpdateTextEditingValue` right? It's part of the TextSelectionDelegate interface.

Reply:
I didn't notice userUpdateTextEditingValue is abstract method. yeah this should be fine.